src/muiComponents/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { forwardRef } from 'react';
+import { default as MaterialUIAvatar, AvatarProps } from '@material-ui/core/Avatar';
+
+// The default element type of MUI's Avatar is 'div' and we don't allow the change of this prop
+type AvatarRef = HTMLElementTagNameMap['div'];
+
+/* eslint-disable verdaccio/jsx-spread */
```

Comment:
I see this pattern being repeated in all `muiComponents`, thus, we can create a `.eslintrc` file at ` src/muiComponents/` level and disable those rules for all files within that folder.

eg: https://github.com/verdaccio/ui/blob/master/test/acceptance/.eslintrc

Reply:
Good idea @juanpicado .. I will do that 🙂